data/moves.ts

```diff
@@ -2849,18 +2849,18 @@ export const Moves: {[moveid: string]: MoveData} = {
 		pp: 20,
 		priority: 0,
 		flags: {},
-		onHit(pokemon) {
-			const noCopycat = [
-				'assist', 'banefulbunker', 'beakblast', 'behemothbash', 'behemothblade', 'belch', 'bestow', 'celebrate', 'chatter', 'circlethrow', 'copycat', 'counter', 'covet', 'craftyshield', 'destinybond', 'detect', 'dragontail', 'dynamaxcannon', 'endure', 'feint', 'focuspunch', 'followme', 'helpinghand', 'holdhands', 'kingsshield', 'matblock', 'mefirst', 'metronome', 'mimic', 'mirrorcoat', 'mirrormove', 'naturepower', 'obstruct', 'protect', 'ragepowder', 'roar', 'shelltrap', 'sketch', 'sleeptalk', 'snatch', 'spikyshield', 'spotlight', 'struggle', 'switcheroo', 'thief', 'transform', 'trick', 'whirlwind',
-			];
+		noCopycat: [
```

Comment:
This isn't what I meant, I meant override `onHit` like everything else; this is a half-refactor because Gen 5 overrides `onHit` again and this just becomes unused extra data. One day these should all be refactored into move flags, but that shouldn't be the goal of this PR.

You also have to override these things for more than just Copycat - any move list where you've added a move that existed in the gen before it, or removed any move has to be accounted for.